tests/testthat/test-tabulate_stats.R

```diff
@@ -0,0 +1,52 @@
+st = calculate_boot_stats(c(1:15), 1000, level = 0.95, seed = 1)
+st_withdist = calculate_boot_stats(c(1:15), 1000, level = 0.95, seed = 1, pass_dist=TRUE)
+
+test_that("Check if error thrown for invalid statistic input", {
+  expect_error(tabulate_stats(list("g"= 1, "m"=2)))
```

Comment:
```suggestion
  expect_error(tabulate_stats(list("g" = 1, "m" = 2)))
```
In R, we should have spaces around "=", can you kindly update it everywhere?